k8s/distributed-chroma/templates/rust-frontend-service.yaml

```diff
@@ -17,6 +17,11 @@ spec:
         - name: rust-frontend-service
           image: "{{ .Values.rustFrontendService.image.repository }}:{{ .Values.rustFrontendService.image.tag }}"
           imagePullPolicy: IfNotPresent
+          livenessProbe:
```

Comment:
i think this needs to be a readiness probe technically - https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes

Reply:
whoops yes